src/v2/Components/FlashBanner/__tests__/FlashBanner.jest.tsx

```diff
@@ -115,7 +115,93 @@ describe("FlashBanner", () => {
       />
     )
 
-    expect(wrapper.text()).toContain("Link expired. Resend verification email.")
+    expect(wrapper.text()).toContain("Link expired.")
+  })
+})
+
+describe("Email confirmation link expired", () => {
+  describe("user can request email confirmation (me?.canRequestEmailConfirmation=true)", () => {
+    it("user is prompted to re-request email confirmation if they can", async () => {
+      const { wrapper } = await getRelayWrapper({
+        data: {
+          me: { canRequestEmailConfirmation: true },
+        },
+        queryString: "?flash_message=expired_token",
+      })
+
+      expect(wrapper.text()).toContain("Link expired.")
```

Comment:
Can we assert upon the larger text that's now displayed. namely "Resend verification email"